File: Daily Challanges/2026-02-February/2026-02-15.py
def get_hill_rating(drop, distance, hill_type):
    type_converter = {
        "Downhill": 1.2,
        "Slalom": 0.9,
        "Giant Slalom": 1.0
    }
    steepness = drop / distance
    adjusted_steepness = steepness * type_converter[hill_type]

    if adjusted_steepness <= 0.1:
        return "Green"
    if adjusted_steepness <= 0.25:
        return "Blue"
    return "Black"


test1 = get_hill_rating(95, 900, "Slalom")
test2 = get_hill_rating(620, 2800, "Downhill")
test3 = get_hill_rating(420, 1680, "Giant Slalom")
test4 = get_hill_rating(250, 3000, "Downhill")
test5 = get_hill_rating(110, 900, "Slalom")
test6 = get_hill_rating(380, 1500, "Giant Slalom")

check1 = "Green"
check2 = "Black"
check3 = "Blue"
check4 = "Green"
check5 = "Blue"
check6 = "Black"

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")
print(f"Test6: {test6 == check6}")

File: Daily Challanges/2026-02-February/2026-02-15.js
function getHillRating(drop, distance, type) {
  const typeConverter = {
        "Downhill": 1.2,
        "Slalom": 0.9,
        "Giant Slalom": 1.0
    }
    const steepness = drop / distance
    const adjustedSteepness = steepness * typeConverter[type]
    
    if(adjustedSteepness <= 0.1){
      return "Green"
    }
    if(adjustedSteepness <= 0.25){
      return "Blue"
    }
    return "Black"
}

const test1 = getHillRating(95, 900, "Slalom");
const test2 = getHillRating(620, 2800, "Downhill");
const test3 = getHillRating(420, 1680, "Giant Slalom");
const test4 = getHillRating(250, 3000, "Downhill");
const test5 = getHillRating(110, 900, "Slalom");
const test6 = getHillRating(380, 1500, "Giant Slalom");

const check1 = "Green";
const check2 = "Black";
const check3 = "Blue";
const check4 = "Green";
const check5 = "Blue";
const check6 = "Black";

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);
console.log(`Test6: ${test6 === check6}`);